CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Fix Dokken configuration for CentOS 8 and Ubuntu 16.04
 - Enable enforce_idempotency checks
 - Remove support for CentOS 6, Ubuntu < 16.04
+- Change segments to Array type #549
```

Comment:
Since I just did a release this morning, can you please move this up into a new ``## Unreleased`` section?

Reply:
I missed that was changed in the merge.  Fixed.